packages/svelte/src/lib/containment/styles/avatar.css

```
@@ -0,0 +1,20 @@
+@reference '$style/index.css';
```

Stylelint (17.7.0)

[error] Unexpected unknown at-rule "@reference" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)